pageserver/src/http/routes.rs

```diff
@@ -48,13 +49,15 @@ struct State {
     auth: Option<Arc<JwtAuth>>,
     allowlist_routes: Vec<Uri>,
     remote_storage: Option<GenericRemoteStorage>,
+    disk_usage_eviction_state: Arc<DiskUsageEvictionState>,
 }
 
 impl State {
     fn new(
         conf: &'static PageServerConf,
         auth: Option<Arc<JwtAuth>>,
         remote_storage: Option<GenericRemoteStorage>,
+        disk_usage_eviction_state: Arc<DiskUsageEvictionState>,
     ) -> anyhow::Result<Self> {
```

Comment:
This does not scale, re: https://github.com/neondatabase/neon/pull/3887/files#r1149560585

Also it doesn't need to return a resullt.

Reply:
Please create a tech-debt follow-up issue and assign to @LizardWizzard , IIRC he's toying around with an alternative HTTP server.

Reply:
Yes, for openapi generation but for management api it's a hard sell to switch.